setup.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import setuptools
+
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name="dataprocmagic",
+    python_requires='>=2.7.0',
+    version="0.1.0",
+    author="Alexis Macaskill",
+    author_email="amacaskill@google.com",
```

Comment:
While I'm supportive of you taking credit for this work, I'm afraid that people will try to email this email address after you've left Google and nobody will check it.

Instead, consider copying the setup function from dataprocspawner, which doesn't specify any maintainer: https://github.com/GoogleCloudDataproc/jupyterhub-dataprocspawner/blob/master/setup.py#L22